lib/active_record/validations.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ActiveRecord::Validations::ClassMethods
+  def validates_associated(association, options = {})
```

Comment:
Reference: https://content.pivotal.io/blog/alias-method-chain-validates-associated-informative-error-message

It's from 2008, but seems to work.